src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -41,6 +42,11 @@ class SubscriberArgs(parser: ArgParser) {
             "--texbe-addr",
             help = "texbe address and port"
     )
+
+    val pandocCompileCommand by parser.storing(
+            "--pandoc-compile",
```

Comment:
I am not sure that we want to have it in the command line. It is a long string which rarely changes, and config file (e.g. a very simple .properties file) is a better place for this.

Reply:
Done. Do I need to move some other variables to config file too?